docs/reference/mapping/types/semantic-text.asciidoc

```diff
@@ -5,4 +5,134 @@
 <titleabbrev>Semantic text</titleabbrev>
 ++++
 
-The documentation page for the `semantic_text` field type.
\ No newline at end of file
+beta[]
+
+The `semantic_text` field type automatically generates embeddings for text
+content using an inference endpoint. 
+
+The `semantic_text` field type specifies an `inference_id` that you can create
+by using the <<put-inference-api>> that will be used to generate embeddings.
+This field type and the <<query-dsl-semantic-query,`semantic` query>> type make
+it simpler to perform semantic search on your data.
+
+Using `semantic_text`, you won't need to specify how to generate embeddings for
+your data, or how to index it. The inference endpoint automatically determines
+the embedding generation, indexing, and query to use.
+
+[source,console]
+------------------------------------------------------------
+PUT my-index-000001
+{
+    "mappings": {
+        "properties": {
+            "inference_field": { 
+                "type": "semantic_text",
+                "inference_id": "my-elser-model"
+	     }
+        }
+    }
+}
+------------------------------------------------------------
+// TEST[skip:TBD]
+
+
+[discrete]
+[[semantic-text-params]]
+==== Parameters for `semantic_text` fields
+
+`inference_id`::
+(Required, string)  
+Inference endpoint that will be used to generate the embeddings for the field.
+Use the <<put-inference-api>> to create the endpoint.
+
+
+[discrete]
+[[infer-endpoint-validation]]
+==== {infer-cap} endpoint validation
+
+The `inference_id` will not be validated when the mapping is created, but when documents are ingested into the index.
+When the first document is indexed, the `inference_id` will be used to generate underlying indexing structures for the field.
+
+
+[discrete]
+[[auto-text-chunking]]
+==== Automatic text chunking
+
+{infer-cap} endpoints have a limit on the amount of text they can process.
+To allow for large amounts of text to be used in semantic search, `semantic_text` automatically generates smaller passages if needed called _chunks_.
+
+Each chunk will include the text subpassage and its corresponding embedding generated from it.
+When querying, the individual passages will be automatically searched for each document, and the most relevant passage will be retrieved as a result.
+
+
+[discrete]
+[[semantic-text-structure]]
+==== `semantic_text` structure
+
+Once a document is ingested, a `semantic_text` field will have the following structure:
+
+[source,console-result]
+------------------------------------------------------------
+"inference_field": {
+  "text": "these are not the droids you're looking for", <1>
+  "inference": {
+    "inference_id": "my-elser-model", <2>
```

Comment:
Note that if you change the `inference_id` above it needs to be changed here as well